crates/types/src/v1alpha1/wasmcloud_host_config.rs

```diff
@@ -104,9 +104,23 @@ pub struct KubernetesSchedulingOptions {
     /// https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/ for valid
     /// values to use here.
     pub resources: Option<WasmCloudHostConfigResources>,
-    #[schemars(schema_with = "pod_schema")]
     /// Any other pod template spec options to set for the underlying wasmCloud host pods.
+    #[schemars(schema_with = "pod_schema")]
+    #[serde(default, skip_serializing_if = "Option::is_none")]
     pub pod_template_additions: Option<PodSpec>,
+    /// Allow for customization of either the wasmcloud or nats leaf container inside of the wasmCloud host pod.
+    pub container_template_additions: Option<ContainerTemplateAdditions>,
```

Comment:
You'll need `#[serde(default, skip_serializing_if = "Option::is_none")]` if you want this to be optional like `pod_template_additions` I think

Reply:
Not in this case, that's handled inside of `ContainerTemplateAdditions` below: https://github.com/wasmCloud/wasmcloud-operator/pull/110/files#diff-07014cdf69e3ecd6540e2c8603771eeb8d0499069a5487edebb37b728f607beeR117-R123

Reply:
As long as it looks good in `kubectl explain` then that works